clients/python/src/model_registry/types/artifacts.py

```diff
@@ -44,13 +65,15 @@ class Artifact(BaseResourceModel, ABC):
 
     Attributes:
         name: Name of the artifact.
-        uri: URI of the artifact.
         state: State of the artifact.
+        description: Description of the artifact.
+        external_id: Customizable ID. Has to be unique among instances of the same type.
     """
 
     name: str | None = None
-    uri: str | None = None
```

Comment:
review note: since only ModelArtifact and DocArtifact has `uri` (ref), then this is pushed down in the specialization classes